test/string-checker.js

```diff
@@ -356,4 +356,24 @@ describe('modules/string-checker', function() {
             });
         }
     });
+
+    describe('getRegisteredPresets', function() {
+        it('returns the configuration\'s registered presets', function() {
```

Comment:
In all fairness, the other delegated methods in string-checker don't have dedicated tests. I'll revert my hide delegate.